dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,34 @@
+package io.dropwizard.auth;
```

Comment:
The file has increased in 34 lines.
<!-- lookout footnote separator -->
_Comment made by the analyzer Dummy._